context/brand-context/HISTORY.md

```diff
@@ -1,5 +1,8 @@
 # History
 
+## 7.0.0 (2021-01-04)
+    * BREAKING: Move font-weight utilities to default context
```

Comment:
Can we mention here that it is moving out of springer, something like
`BREAKING: font-weight utilities moved from SPRINGER to DEFAULT context`
Will help with scanning

Reply:
Most certainly 👍